test/examples/src/man/tutorial.md

```diff
@@ -403,3 +403,33 @@ Hello World!
 Written in <a href="https://fonts.google.com/specimen/Nanum+Brush+Script">Nanum Brush Script.</a>
 </div>
 ```
+
+## Handling of `text/latex`
+
+You can define a type that has a `Base.show` method for the `text/latex` MIME:
+
+```@example showablelatex
+struct LaTeXEquation
+    code :: String
+end
+Base.show(io, ::MIME"text/latex", latex::LaTeXEquation) = write(io, latex.code)
+nothing # hide
+```
+
+In an `@example` or `@eval`block, it renders as an equation:
+
+```@example showablelatex
+LaTeXEquation("x^2")
+```
```

Comment:
This example, as is, will not render render as an equation anymore I think? Because this is the fallback situation? Could maybe be replaced with something like:

```suggestion
```@example showablelatex
LaTeXEquation("Foo $x^2$ bar.")
```
```